src/utils/SpoonacularRequests.ts

```diff
@@ -16,6 +13,21 @@ export const fetchData = async() => {
     }
     return searchString;
   }
-  
+  const call: any = await fetch(`https://api.spoonacular.com/recipes/findByIngredients?apiKey=ec8da5b51af4425698dd91381535afff&ingredients=${ingredientSearch(ingredientList)}`);
+
+  const response = await call.json()
+
+  const onCallHandler = () => {
+    let title: string;
+    let id: number;
+    let image: string;
+    for(let i = 0; i < 5; i++) {
```

Comment:
instead of 5 in the loop statically make it a parameter then add 5 as the argument